Cargo.toml

```diff
@@ -7,9 +7,45 @@ description = "BoringSSL crypto provider for quinn"
 keywords = ["quic"]
 categories = ["network-programming", "asynchronous"]
 edition = "2021"
-rust-version = "1.59"
 
 [badges]
 maintenance = { status = "passively-maintained" }
 
+[features]
+fips = ["boring/fips", "boring-sys/fips"]
+
 [dependencies]
+boring = "2.1.0"
+boring-sys = "2.1.0"
+bytes = "1"
+foreign-types-shared = "0.3.1"
+lazy_static = "1"
```

Comment:
Recommend using once_cell instead of lazy_static.

Reply:
Done.